events/sr2017/kickstart.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: news
+title: SR2017 Kickstart
+permalink: /events/sr2017/kickstart
+date: 2016-10-03
+---
+
+The venues for Student Robotics 2017 kickstart are:
+
+- University of Southampton
+- University of Bristol
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
+
+Below is an incomplete table of venue-dependent information, which will be completed very soon.
+
+| Venue                     | How to get there                                     | Requires a laptop? | Nearby Food Locations | Finish Time |
```

Comment:
The start time is the same for all venues, this table is of venue-dependent information. I'd argue it forces them to actually read the rest of the information in the document